pdr_backend/lake/gql_data_factory.py

```diff
@@ -156,6 +156,10 @@ def _do_subgraph_fetch(
 
         buffer_df = pl.DataFrame([], schema=table.df_schema)
 
+        PersistentDataStore(self.ppss.lake_ss.lake_dir).create_table_if_not_exists(
+            get_table_name(table.table_name, TableType.TEMP), table.df_schema
+        )
```

Comment:
This isn't needed.
- fetch will return and table.append_to_storage() will be called
- this in returns call _append_to_db()
- this calls `PersistentDataStore(self.base_path).insert_to_table(data, table_name)`
- insert_to_table check if the table doesn't exist, and creates it if needed

Reply:
I added this logic because if the data is empty, we do not create the table and it causes error on the bronze process

Reply:
thanks for sharing this, it makes sense
let's make sure we integrate this